index.html

```diff
@@ -7,7 +7,7 @@
     <title>Document</title>
 </head>
 <body>
-    <h1>Javascript Basics</h1>
+    <h1>Javascript Basics Test Git</h1>

```

Comment:
OK. It's fine

Reply:
Need to replace 
`<h1>Javascript Basics Test Git</h1>`
to 
<h2>Javascript Basics Test Git</h2>

Reply:
resolved